Dockerfile

```
@@ -0,0 +1,27 @@
+ARG PYTHON_VERSION=3.8
+FROM python:${PYTHON_VERSION} AS base
+
+WORKDIR /app
+
+COPY ./accounts ./accounts
+COPY ./api ./api
+COPY ./lists ./lists
+COPY ./todolist ./todolist
+COPY manage.py requirements.txt ./
```

Copy files this way will simplify the Dockerfile. If you don't want to copy useless files inside container, use [.dockerignore](https://docs.docker.com/build/concepts/context/#dockerignore-files)
```suggestion
COPY . .
```